cmp/tab/TabPane.js

```diff
@@ -39,7 +39,6 @@ export class TabPane extends Component {
 
         return frame({
             display: isActive ? 'flex' : 'none',
-            margin: 4,
```

Comment:
From what I could tell, this was always getting blown away by some blueprint css with an `!important` flag setting the margin to 0